kore/test/Test/Kore/OnePath/Step.hs

```diff
@@ -199,148 +199,147 @@ test_onePathStrategy =
                 [ _actual
                 ]
             )
-     , testCase "Differentiated axioms" $ do
-         -- Target: constr11(a)
-         -- Coinductive axiom: constr11(a) => g(a)
-         -- Coinductive axiom: constr11(b) => f(b)
-         -- Normal axiom: constr11(a) => g(a)
-         -- Normal axiom: constr11(b) => g(b)
-         -- Normal axiom: constr11(c) => f(c)
-         -- Normal axiom: constr11(x) => h(x)
-         -- Normal axiom: constr10(x) => constr11(x)
-         -- Start pattern: constr10(x)
-         -- Expected:
-         --   (f(b) and x=b)
-         --   or (f(c) and x=c)
-         --   or (h(x) and x!=a and x!=b and x!=c )
-         actual <-
-             runOnePathSteps
+    , testCase "Differentiated axioms" $ do
+        -- Target: constr11(a)
```

Comment:
This test was hard for me to understand because the `Target` and the `Start pattern` are separated. Maybe this could say,

```.hs
-- Goal: constr10(x) => constr11(a)
```

Reply:
https://github.com/kframework/kore/pull/979